tasks/src/test/java/org/example/aProblemAboutEquation/AProblemAboutEquationTest.java

```diff
@@ -0,0 +1,10 @@
+package org.example.aProblemAboutEquation;
+
+import org.junit.Test;
+
+class AProblemAboutEquationTest {
+
+/**
+ * Klasa praktycznie nietestowalna
```

Comment:
jezeli jest nietestowalna to najlepiej wyrzucic ta klase w ogole z kodu. Bo ona nic nie daje. 
Lub wyrzucic implementacje z gita ktorej sie testowac nie da. Bo kod nietestowalny to kod nieakceptowalny :D